dockerfiles/scripts/cron_job_dump_ledger.sh

```diff
@@ -9,31 +9,36 @@ echo "done sleeping"
 
 # retry getting the staking ledger until the node is fully up and the command returns exit code 0
 while true; do
-  mina ledger export staking-epoch-ledger > staking_epoch_ledger.txt
-  if [ "$?" -eq 0 ] && [ "$(cat staking_epoch_ledger.txt)" != "Ledger not found: current staking ledger not available" ]; then
+  mina ledger export staking-epoch-ledger > staking_epoch_ledger.json
```

Comment:
could you check if the daemon is synced and then get the staking ledgers (instead of waiting for 8 mins)? You can check if sync status is `Synced` and `Block height = `Max observed block height`

Reply:
Wouldn't I still have to wait and poll to see if the status is `Synced`?  If it isn't synced I'd just have to wait more and check again wouldn't I?

Also, is this critical?  The cron job is already deployed and working ok, this PR is mostly just for archival purposes.